### DISCUSSION TITLE
native stack does not allow gestureDirection, unlike normal stack

### DISCUSSION THREAD

**Discussion by fai9al7dad** [software-mansion/react-native-screens#1196] [Ideas]
I'm currently trying to 'swipe-from-right' to close a stack.

previously on the normal stack i would go to the stack navigator and pass gestureDirection :'horziontal-inverted' on screen options. is there a way that same effect could happen in native stack ?

**Comment by WoLewicki** [comment]
Can you try if setting `direction: 'rtl'` in `screenOptions` is what you need (https://github.com/software-mansion/react-native-screens/tree/main/native-stack#direction)? Be aware that it will also change the position of your header's elements like back button, which seems reasonable.

**Reply by fai9al7dad** [reply]
Yes, that worked perfectly 👌

Thanks
